changelogs/CHANGELOG-v0.3.0.md

```diff
@@ -0,0 +1,38 @@
+The v0.3.0 release is our second official minor release of the the bpfd project :tada
```

Comment:
Do we need a disclaimer at the bottom that there were a lot of API changes and we do not claim backwards compatibility?

Reply:
Sounds like a good idea.

Reply:
+1 sounds good